.vscode/pnpify/eslint/bin/eslint.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const {createRequire, createRequireFromPath} = require(`module`);
+const {dirname, resolve} = require(`path`);
```

Comment:
`dirname` is included in all the SDKs but none of them actually use it anymore

Reply:
Fixed 👍